File: OpenEngine-Editor/src/FlappyBird/Camera.cpp
#include "oepch.h"
#include "Camera.h"

namespace OpenEngine {

	void CameraScript::OnCreate()
	{
		//m_Player = GetEntityByName("Player");
	}

	void CameraScript::OnUpdate(Timestep ts)
	{
		//GetComponent<TransformComponent>().Translation.x += 2.0f * ts;
	}

	void CameraScript::OnDestroy()
	{

	}

}

File: OpenEngine-Editor/SandboxProject/Assets/Scripts/Source/Camera.cs
using OpenEngine;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sandbox
{
    public class Camera : Entity
    {
        public Entity Player;

        public void OnCreate()
        {
            Player = GetEntityByName("Player");
        }

        public void OnUpdate(float ts)
        {
            Translation = Player.Translation;
        }
    }
}
